cmd/kubeadm/app/phases/certs/certs_test.go

```diff
@@ -739,15 +739,25 @@ func TestNewCSR(t *testing.T) {
 		t.Errorf("invalid signature on CSR: %v", err)
 	}
 
-	assert.ElementsMatch(t, certConfig.Organization, csr.Subject.Organization, "organizations not equal")
+	sort.Strings(certConfig.Organization)
```

Comment:
This modifies the object under test, the old line did not. I don't know if this is important, but this is a common-ish way to get a bug in a test.